authorization-common/src/main/java/com/sequenceiq/authorization/service/UtilAuthorizationService.java

```diff
@@ -60,23 +81,24 @@ public CheckResourceRightsV4Response getResourceRightsResult(CheckResourceRights
         Multimap<ImmutablePair<String, RightV4>, AuthorizationProto.RightCheck> resourceRightsChecks = LinkedListMultimap.create();
```

Comment:
I know that possibly I created this method, but maybe it is worth to split into multiple methods, looks hard to understand it

Reply:
Sure.